css/style.css

```diff
@@ -1,365 +1,392 @@
-/* Color Scheme. Please use var() to set colours. This will make site-wide changes later much easier */
-:root{
-    --primary-grey:#222831;
```

Comment:
By using color names, it defeats the purpose of having a _global variable_. What if we wanted to change from grey to blue? Would we change this and all references to blue?

Reply:
Yes good point, accepted change